.evergreen/run-x509-tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+source "${PROJECT_DIRECTORY}/.evergreen/init-node-and-npm-env.sh"
+
+export SSL_KEY_FILE=$DRIVERS_TOOLS/.evergreen/x509gen/client.pem
+export SSL_CA_FILE=$DRIVERS_TOOLS/.evergreen/x509gen/ca.pem
+export SSL_KEY_FILE_EXPIRED=$DRIVERS_TOOLS/.evergreen/x509gen/expired.pem
+export SSL_KEY_NO_USER=$DRIVERS_TOOLS/.evergreen/x509gen/crl.pem
+
+export SUBJECT=$(openssl x509 -subject -nameopt RFC2253 -noout -inform PEM -in $SSL_KEY_FILE)
```

Comment:
IIUC if openssl fails to run we won't get an error code from it because the export step will always return 0
https://www.shellcheck.net/wiki/SC2155